packages/notebook-extension/src/index.ts

```diff
@@ -2347,29 +2353,36 @@ function addCommands(
   });
 
   commands.addCommand(CommandIDs.toggleRenderSideBySide, {
-    label: trans.__('Render Side-by-side'),
+    label: trans.__('Toggle Side-by-side rendering setting'),
```

Comment:
```suggestion
    label: trans.__('Render Side-by-side by Default'),
```

Or some variation? Like I don't feel like it's much better, but the current version is not distinct enough from the per-notebook command IMO.

Reply:
The `toggleRenderSideBySide` is responsible to toggle the setting. I am not sure that `Render Side-by-side by Default` expresses that action, especially that the sentence remains the same whatever the setting state (true/false) is.

Reply:
Yes, my proposal is not great. But I do feel like "toggle setting" will not be obvious to the user. Maybe something like "Toggle default notebook layout: side-by-side"?

Reply:
I have no strong opinion on that. Maybe input from @fcollonval or @jtpio would help.

Reply:
Another option could be `Enable Side-by-side Rendering by Default`?

Reply:
I have removed that command which had been there in the previous iteration of the side-by-side, but looking closer, was not changing for real the user setting, so was not doing the job it was supposed to do.

Also we have generalized the the notion of rendering layout, so it become difficult anyway to find a good fit for that commande.

Reply:
Earlier I proposed to use `Render Side-by-Side` (using the :white_check_mark: mark to distinguish the status) to be aligned with for example `Show line numbers` (that is also per document) or `Wrap words`.

Reply:
Hi, thx for the help here. To wrap up, we have a setting `renderingLayout` which can be `default` or `side-by-side` and we are talking about 2 commands

- The one that changes toggle the layout of the current notebook and is available via the menu. The name is now `Toggle Side-by-side Rendering`. I can change it to `Render Side-by-Side` with the ✅.
- The one that was there before and I have removed which was supposed to toggle the user setting. The former implementation had just a in-memory flag that was used to apply the toggle the rendering on all the notebooks, but was note changing the user setting. I don't think a command can, or should change a user setting. This is why I have dropped it